zoom_monitor.py

```diff
@@ -10,6 +10,7 @@
                                         NoSuchElementException)
 from selenium.webdriver import Chrome
 from selenium.webdriver.chrome.options import Options
+#from selenium.webdriver.common.action_chains import ActionChains
```

Comment:
If this is commented out, does it need to be here?

Reply:
Ugh, this is from the rebase. Thanks for catching it!